velox/type/tests/SubfieldTest.cpp

```diff
@@ -20,12 +20,11 @@
 using namespace facebook::velox::common;
 
 std::vector<std::unique_ptr<Subfield::PathElement>> tokenize(
-    const std::string& path,
-    const std::shared_ptr<Separators>& separators = Separators::get()) {
+    const std::string& path) {
   std::vector<std::unique_ptr<Subfield::PathElement>> elements;
-  Tokenizer tokenizer(path, separators);
-  while (tokenizer.hasNext()) {
-    elements.push_back(tokenizer.next());
+  auto tokenizer = Tokenizer::getInstance(path);
```

Comment:
Maybe add a unit test to register a dummy tokenizer, and test its customized behavior.

Reply:
Sure, let me add one, thanks.

Reply:
Thanks, added.